moderation-commands/clean.md

```diff
@@ -25,18 +25,25 @@ For more information on editing permission requirements for specific users/roles
 
 ### Options
 
-* `amount`: <TODO: describe>
-* `user`: <TODO: describe>
-* `starts-with`: <TODO: describe>
-* `ends-with`: <TODO: describe>
-* `contains`: <TODO: describe>
-* `bots`: <TODO: describe>
-* `system`: <TODO: describe>
-* `preserve-pins`: <TODO: describe>
-* `no-trace`: <TODO: describe>
+* `amount`: The amount of messages you want to clean (The default is 100)
+* `user`: The user you want to do delete the messages of
+* `starts-with`: Delete messages that start with this value
+* `ends-with`: Delete messages that end with this value
+* `contains`: Delete messages that contain this value
+* `bots`: Whether or not to delete messages sent by bots (The default is false)
+* `system`: Whether or not to delete system messages. System messages are messages sent by Discord, such as join or pin messages
+![An example of a system message](/.gitbook/assets/SystemMessage.png "System Message Example")
+* `preserve-pins`: If true, pinned messages will not be deleted
+* `no-trace`: If true, the confirmation message will only be visible to you
 
 ## Logs
 
-* `CHANNEL_CLEAN`: <TODO: describe>
+* `CHANNEL_CLEAN`: MoonlightBot will send two things: an HTML message that, when opened, displays the deleted messages, and a message detailing where messages were cleaned, how many were cleaned, who cleaned them, and the filters applied
 
-For more information on setting up those logs, refer to the [log setup tutorial](<linkToLogTutorial>)
\ No newline at end of file
+![An example of logs sent by the bot after a clean command execution](/.gitbook/assets/CleanLogs.png "Clean Logs")
+
+{% hint style="info" %}
+To open an HTML file, download it and open the file explorer. Double-click the file, and your computer will open a new tab in your browser to display the executed code
```

Comment:
this doesn't execute anything html is just to have a fancy visualization of the message, needs rewording